soc/xtensa/intel_adsp/tools/README

```diff
@@ -0,0 +1,116 @@
+This documentation descripts how to run the intel_adsp_cavs board.
```

Comment:
Please move this into rst and make sure it's included somewhere in the generated documentation.

Reply:
I updated it to the rst file and move this to boards/xtensa/intel_adsp_cavs15/doc/generic_intel_adsp.rst. Thanks for the review.